### ISSUE TITLE
sadly, enable cors behavior is incorrect

### ISSUE THREAD

**Issue by dominictarr** [Raynos/fake-api-gateway-lambda#2]
The enable cors functionality in this module is how I wish api-gateway worked -- it sets the header on the response, which is sent to the client, even if the lambda doesn't set it. however, this isn't actually how aws lambda works. When I was trying to make cors work with lambda it only worked when I explicitly set the cors headers in the lambda.

**Comment by Raynos** [comment]
Right, this behavior should change to only handle the `OPTIONS` method.
